.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "env": {
+    "es6": true,
+    "node": true
+  },
+  "extends": ["eslint:recommended"],
+  "globals": {
```

Comment:
What are these for? I can't find any documentation on these in ESLint.

Reply:
Actually I was finding `"eslint:recommended"` as it was mentioned in the issue. I used it because of this [link]( https://stackoverflow.com/questions/33162047/where-can-i-find-the-rule-definitions-for-eslintrecommended)